common/src/main/java/com/microsoft/identity/common/internal/net/ObjectMapper.java

```diff
@@ -96,4 +103,34 @@ public static String serializeObjectToFormUrlEncoded(Object object) throws Unsup
         return builder.toString();
     }
 
+    public static Map<String, String> extraQueryString(final String extraQueryString) {
```

Comment:
deserializeQueryStringToMap for method name?  queryString for parameter name?  Rather than this lower level parsing... do you want to use this instead: https://developer.android.com/reference/android/net/Uri.html#getQueryParameterNames()